lib/rules/require-dollar-before-jquery-assignment.js

```diff
@@ -75,6 +75,41 @@ module.exports.prototype = {
             var varName;
             var right;
 
+            function checkIfVarNameShouldStartWithDollar(left, varName, right) {
+
+                if (varName.indexOf('$') === 0 || varName.indexOf('_$') === 0) {
+                    return;
+                }
+
+                if (!right || right.type !== 'CallExpression') {
```

Comment:
Is there a quick code sample where the right is not there? Either a test or comment would be good. The test is preferable though